truss/templates/docker_server/proxy.conf.jinja

```diff
@@ -0,0 +1,31 @@
+server {
+    # We use the proxy_read_timeout directive here (instead of proxy_send_timeout) as it sets the timeout for reading a response from the proxied server vs. setting a timeout for sending a request to the proxied server.
+    listen 8080;
+
+    # Liveness
+    location / {
```

Comment:
remind me the function of readiness vs. liveness?

Reply:
i've had to look [this](https://chatgpt.com/share/e/ed536f84-c582-4dd0-8ae6-a1e8318c19d9) up a few times.